### PR TITLE
minor edits to answer questions generated following installation guide

### PR DESCRIPTION
**Pull request by joelong01**
- updated URIs to point to actual installation directories instead of landing pages
- updated instructions to specify what options to pick
- warned the user of mistakes I made so they can avoid them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +50,17 @@ An example cross platform middle tier api is provided as part of the solution, i
     - [Installing the Linux Subsystem on Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
 3. Docker
     - [Install on Mac](https://docs.docker.com/docker-for-mac/install/)
-    - [Install on Windows](https://docs.docker.com/docker-for-windows/install/)
+    - [Install on Windows](https://store.docker.com/editions/community/docker-ce-desktop-windows)
+        - Select the "Use Windows containers ..." option
+        - After rebooting, you will be prompted to login to Docker.  You can create a personal username and pasword here.
```
Comment (billpratt) [inline README.md]: Logging in should be optional, at least on Mac it is.

File: README.md [hunk 2]
```diff
@@ -67,13 +69,17 @@ An example cross platform middle tier api is provided as part of the solution, i
...
         ```powershell
         Install-Script -Name install-kubectl -Scope CurrentUser -Force
-        install-kubectl.ps1 [-DownloadLocation <path>]
+        install-kubectl.ps1 -DownloadLocation <path>
+            - where <path> is the location you wish to install it. 
+            - you may be prompted to install a new version of nuget.  do it. 
         ```
 6. ACS Engine
     - [Download and install ACS Engine](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md#install)
+        - be sure and put (or add) acs-engine.exe to your $PATH
```
Comment (billpratt) [inline README.md]: 👍

**Comment by joelong01** [comment]
Below >>

File: README.md (hunk 2 above)
Comment (wesyao) [inline README.md]: Should we add any comments for other common OS like OSX or linux?

File: README.md (hunk 1 above)
Comment (billpratt) [reply README.md]: @joelong01 Maybe add something like "After rebooting, you will be prompted to optionally login to Docker"

File: README.md (hunk 2 above)
Comment (jobanputra) [reply README.md]: The first two lines on the page the url take you to are (emphasis mine):
>Binary downloads for the latest version of acs-engine for are available here. Download acs-engine for your operating system. **Extract the binary and copy it to your $PATH**.

I don't see why we need to repeat the instructions here.

Comment (jobanputra) [reply README.md]: The first two lines on the page the url take you to are (emphasis mine):
>Binary downloads for the latest version of acs-engine for are available here. Download acs-engine for your operating system. **Extract the binary and copy it to your $PATH**.

I don't see why we need to repeat the instructions here.

**Comment by joelong01** [comment]
The reason I put it there is because it was the nth step where you installed something that downloaded and ran setup and then on this step we just downloaded an EXE. It is easy to miss when you are just following the readme.

joe

**Review by aaron-schnieder — APPROVED** [review]

**Comment by wesyao** [comment]
Is this PR stale now?

**Pull request closed without merging**